tools/system_libs.py

```diff
@@ -232,9 +231,8 @@ def create_ninja_file(input_files, filename, libname, cflags, asflags=None, cust
       # Resolve duplicates by appending unique.
       # This is needed on case insensitive filesystem to handle,
       # for example, _exit.o and _Exit.o.
-      object_basename = shared.unsuffixed_basename(src)
-      if case_insensitive:
-        object_basename = object_basename.lower()
+      # Always apply it to produce same libraries on different filesystems.
```

Comment:
How about "This is done even on case sensitive filesystem so that builds are reproducible across platforms".

Also, can you reflow these 4 lines or comments wrapping at 80 columns.